ogticrd/covid-safe-paths
# Feat: legal warning spanish version

#### Description:


I translated to Spanish the legal warning 

#### Linked issues:


Fix: [153](https://trello.com/c/kaHwbhzG/153-cambiar-aviso-legal-a-espa%C3%B1ol)

#### Screenshots:


![image](https://user-images.githubusercontent.com/43874147/84674421-e4162380-aef8-11ea-8d05-9c9b9b2b62e9.png)


#### How to test:


- `git pull && git checkout Feat/153-aviso-legal-spanish-version`
- Head over to **Legal** screen and check the legal warning is in Spanish if the app is in Spanish version
